source/blender/gpu/intern/gpu_material.c

```diff
@@ -2779,6 +2779,16 @@ float *GPU_lamp_dynpersmat(GPULamp *lamp)
 	return &lamp->dynpersmat[0][0];
 }
 
+float *GPU_lamp_get_viewmat(GPULamp *lamp)
```

Comment:
const at the end of the function is for C++ to say that the object will not be modified. Here there's no object but only a returned value, this value should be const to avoid modify it so:
```
const float *GPU_lamp_get_viewmat(GPULamp *lamp);
```